test/infrastructure/docker/docker/kind_manager.go

```diff
@@ -17,24 +17,32 @@ limitations under the License.
 package docker
 
 import (
+	"context"
 	"fmt"
+	"io"
 	"net"
 	"os"
 	"strings"
 
+	dockerTypes "github.com/docker/docker/api/types"
```

Comment:
Last nit from my side. Afaik camel case is not used for import aliases (just lower case).

Reply:
Got it - picked that up from somewhere else, but I also prefer lower case.

Reply:
Yup, I also had that problem from time to time :)